.github/workflows/welcome.yml

```diff
@@ -0,0 +1,33 @@
+name: welcome
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  welcome-message:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: Beep boop beep... I am a real human... for real.\n\n
+          Welcome to the RIOT community, we will try to address this as soon as we can.
+          As I mentioned before, we are human and sometimes we get distracted with life.
+          I can try to ping someone like this.\n\n
+          ping @MrKevinWeiss\n\n
+          Feel free to do the same for other [maintainers](https://github.com/orgs/RIOT-OS/teams/maintainers/members) if things are taking a while.
+        pr-message: Beep boop beep... I am a real human... for real.\n\n
+          Welcome to the RIOT community, we will try to review this as soon as we can.
+          As I mentioned before, we are human and sometimes we get distracted with life.
+          I can try to ping someone like this.\n\n
+          ping @MrKevinWeiss\n\n
+          Feel free to do the same for other [maintainers](https://github.com/orgs/RIOT-OS/teams/maintainers/members) if things are taking a while.
```

Comment:
If this is the same message as for `issue-message`, could you try setting it as an environment variable and use it here and for `issue-message` instead?

```yaml
    - uses: actions/first-interaction@v1
      env:
        GREETER_MESSAGE: >
          Beep boop beep... I am a real human... for real.\n\n
          Welcome to the RIOT community, we will try to address this as soon as we can.
          As I mentioned before, we are human and sometimes we get distracted with life.
          I can try to ping someone like this.\n\n
          ping @MrKevinWeiss\n\n
          Feel free to do the same for other [maintainers](https://github.com/orgs/RIOT-OS/teams/maintainers/members) if things are taking a while.
      with:
        repo-token: ${{ secrets.GITHUB_TOKEN }}
        issue-message: ${{ env.GREETER_MESSAGE }}
        pr-message: ${{ env.GREETER_MESSAGE }}
```